spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+require './lib/vehicle'
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+
+
+RSpec.describe VehicleFactory do
+  describe '#initialize' do
+    it 'instance of factory' do
+      factory = VehicleFactory.new
+
+      expect(factory).to be_an_instance_of(VehicleFactory)
+    end
+  end
+
+  describe '#ev_registrations' do
+    it 'has data' do
```

Comment:
This test is really testing `#create_vehicles`. See if you can be more explicit with your naming.